slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -32,6 +32,9 @@ DEFINE_int64(
 DEFINE_string(input_file_pattern, "", "Input file pattern");
 DEFINE_bool(delete_processed_data, false,
             "Deletes data after it has been processed");
+DEFINE_bool(use_live_data, false,
```

Comment:
Default value for flag is currently set to false, although since it is mandatory this should never affect operation)